ggd-recipes/GRCh37/gnomad.yaml

```diff
@@ -1,64 +1,30 @@
-# - filters only PASS variants (segdup, decoy are retained)
-# - removes many INFO fields to reduce file size: controls_, hist, non_, using bcftools annotate instead of vt rminfo, because it failed randomly on chr1
-# - merges all chr into a single file
-# wget is separated from the processing because wget in the pipe fails randomly (on some systems?): it comes out when downloading many files.
 ---
 attributes:
   name: gnomad
-  version: 2.1
+  version: 2.1.1
 recipe:
   full:
     recipe_type: bash
     recipe_cmds:
       - |
-        url_prefix=http://ftp.ensemblorg.ebi.ac.uk/pub/data_files/homo_sapiens/GRCh37/variation_genotype/gnomad/r2.1/genomes/
-        vcf_prefix=gnomad.genomes.r2.1.sites.chr
-        
-        ref=../seq/GRCh37.fa
+        url_prefix=https://storage.googleapis.com/gnomad-public/release/2.1.1/vcf/exomes/
+        vcf_file=gnomad.exomes.r2.1.1.sites.vcf.bgz
         mkdir -p variation
         export TMPDIR=`pwd`
-
         gnomad_fields_to_keep_url=https://gist.githubusercontent.com/naumenko-sa/d20db928b915a87bba4012ba1b89d924/raw/cf343b105cb3347e966cc95d049e364528c86880/gnomad_fields_to_keep.txt
         wget --no-check-certificate -c $gnomad_fields_to_keep_url
-
-        # no chr Y for gnomad genomes
-        for chrom in $(seq 1 22;echo X)
-        do
-          vcf=${vcf_prefix}${chrom}_noVEP.vcf.gz
-          vcf_url=${url_prefix}${vcf}
-
-          wget -c $vcf_url
-          wget -c $vcf_url.tbi
-
-          fields_to_keep="INFO/"$(cat gnomad_fields_to_keep.txt | paste -s | sed s/"\t"/",INFO\/"/g)
-          gunzip -c $vcf | gsort -m 3000 /dev/stdin $ref.fai | bcftools view -f PASS -Ov | bcftools annotate -x "^$fields_to_keep" -Ov | vt normalize -r $ref -n - | vt uniq - | grep -v "##contig="| bgzip -c >  variation/gnomad_genome.chr${chrom}.vcf.gz
-       
-          rm $vcf
-          rm $vcf.tbi
-
-          tabix variation/gnomad_genome.chr${chrom}.vcf.gz
-        done
-
-        cd variation
-        bcftools concat gnomad_genome.chr1.vcf.gz gnomad_genome.chr2.vcf.gz gnomad_genome.chr3.vcf.gz gnomad_genome.chr4.vcf.gz gnomad_genome.chr5.vcf.gz gnomad_genome.chr6.vcf.gz \
-        gnomad_genome.chr7.vcf.gz gnomad_genome.chr8.vcf.gz gnomad_genome.chr9.vcf.gz gnomad_genome.chr10.vcf.gz gnomad_genome.chr11.vcf.gz gnomad_genome.chr12.vcf.gz \
-        gnomad_genome.chr13.vcf.gz gnomad_genome.chr14.vcf.gz gnomad_genome.chr15.vcf.gz gnomad_genome.chr16.vcf.gz gnomad_genome.chr17.vcf.gz gnomad_genome.chr18.vcf.gz \
-        gnomad_genome.chr19.vcf.gz gnomad_genome.chr20.vcf.gz gnomad_genome.chr21.vcf.gz gnomad_genome.chr22.vcf.gz gnomad_genome.chrX.vcf.gz -Ov | bgzip -c > gnomad_genome.vcf.gz
-
-        tabix -f -p vcf gnomad_genome.vcf.gz
-        tabix -f -p vcf --csi gnomad_genome.vcf.gz
-
-        rm gnomad_genome.chr*.vcf.gz
-        rm gnomad_genome.chr*.vcf.gz.tbi
-
-        cd ..
+        vcf=${vcf_prefix}${vcf_file}
```

Comment:
Yup @kokyriakidis is right, looks like it should be url_prefix here.

Reply:
Actually it is better to change url_prefix to vcf_prefix variable earlier in the file because this is how it is called in the other gnomad files e.x. for hg38.